src/features/smartSearch/l10n/messageIds.ts

```diff
@@ -301,6 +305,17 @@ export default makeMessages('feat.smartSearch', {
         sent: m('been sent'),
       },
     },
+    joinForm: {
+      anyForm: m('any join form'),
```

Comment:
(if you decide it should be "Join Form" in the prev comment, change here as well)

Reply:
Going with lower case.